FLEDGE.md

```diff
@@ -924,16 +964,33 @@ The output of `generateBid()` contains the following fields:
     browser to select only some of the returned adComponents in order to help
     make the ad k-anonymous. See [Flexible Component Ad Selection Considering k-Anonymity](#341-flexible-component-ad-selection-considering-k-anonymity)
     for more details.
+*   selectedBuyerAndSellerReportingId: (optional) A string from the interest
+    group's ad's `selectableBuyerAndSellerReportingIds` array; if it's not in
+    the array, the bid is ignored.  If present and jointly k-anonymous with
+    `buyerAndSellerReportingId`, `buyerReportingId`, the interest group owner,
+    bidding script URL, and render URL, then it will be presented to
+    `reportWin()` and `reportResult()`. See
+    [Reporting IDs](#54-reporting-ids-in-reporting) for more details.
+*   selectedBuyerAndSellerReportingIdRequired: (optional) A boolean that when `true`,
+    indicates this bid should be thrown away if `selectedBuyerAndSellerReportingId`
+    is not present or if the k-anonymity check on it fails. Setting this to `true`
+    indicates that reporting of this bid would not operate correctly were the
+    `selectedBuyerAndSellerReportingId` not presented to `reportWin()` and
+    `reportResult()`. See [Reporting IDs](#54-reporting-ids-in-reporting)
+    for more details.
 
 In case returning multiple bids is supported by the implementation in use,
 `generateBid` may also return up to `browserSignals.multiBidLimit` valid bid
 objects of the format above in an array.
 
 Note: Chrome currently imposes an internal limit of 100 for the length of returned bids sequences.
 
-If none of the produced bids pass the k-anonymity checks, `generateBid` will be
+If none of the produced bids pass the k-anonymity checks (the check on the
+render `url` or, when `selectedBuyerAndSellerReportingIdRequired` is `true`, the
+check on `selectedBuyerAndSellerReportingId`), `generateBid` will be
```

Comment:
when `selectedBuyerAndSellerReportingIdRequired` is `true`, I recall the check will be on `selectedBuyerAndSellerReportingId`, and 'buyerAndSellerReportingId' (if present), and 'buyerReportingId' (if present). Is that right?

Reply:
Well, we removed `selectedBuyerAndSellerReportingIdRequired` and Orr's suggested wording above includes `buyerAndSellerReportingId` and `buyerReportingId` now.